### ISSUE TITLE
Docker build failure when running with deploy.sh

### ISSUE THREAD

**Issue by KaminariOS** [ASUCICREPO/PDF_Accessibility#24]
Log:

```
PDFAccessibility: fail: docker build --tag cdkasset-987b1f0cb24893a79e39568d2b3245bc2051b9d924dea8ee873b9a0f04b41edb --platform linux/amd64 . exited with error code 1: #0 building with "default" instance using docker driver

```

**Comment by KaminariOS** [comment]
My organization does not allow PVC.

Is there a way to circumvent this?
